doc/releases/migration-guide-3.7.rst

```diff
@@ -170,6 +170,13 @@ Networking
   one IPv4 address / network interface, the netmask must be specified
```

Comment:
Please change the migration guide commit subject to
```
 doc: migration-guide-3.7: Add wifi shell connect cmd info
```
because we are actually changing the documentation and not the wifi stuff in this commit.

Reply:
Done